libs/sdk-ui-charts/src/charts/headline/CoreHeadline.tsx

```diff
@@ -65,8 +81,11 @@ export class HeadlineStateless extends React.Component<Props> {
 }
 
 /**
- * NOTE: exported to satisfy sdk-ui-ext; is internal, must not be used outside of SDK; will disapppear.
- *
  * @internal
  */
-export const CoreHeadline = withEntireDataView(HeadlineStateless);
+const CoreHeadline = withEntireDataView(HeadlineStateless);
+
+/**
+ * NOTE: exported to satisfy sdk-ui-ext; is internal, must not be used outside of SDK; will disapppear.
+ */
+export { CoreHeadline, ICoreHeadlineExtendedProps };
```

Comment:
Hi @xMort ,

As I understand it, there is no need to tag it as @internal, because we have already tagged each export element here.
https://github.com/gooddata/gooddata-ui-sdk/pull/3927/files#diff-f940fe2ed2a939f3da6fac6cf660b81557f23b9593ae060bee220b0c28041d46R16
https://github.com/gooddata/gooddata-ui-sdk/pull/3927/files#diff-f940fe2ed2a939f3da6fac6cf660b81557f23b9593ae060bee220b0c28041d46R84

following that, the API was constructed:
https://github.com/gooddata/gooddata-ui-sdk/pull/3927/files#diff-48f53b057b9b9ff91a76854fa633419e7f0aeb9ea2d66f915de8f14b76e6ae22R95
https://github.com/gooddata/gooddata-ui-sdk/pull/3927/files#diff-48f53b057b9b9ff91a76854fa633419e7f0aeb9ea2d66f915de8f14b76e6ae22R348